CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+# Contributing to Bandit
+Thanks for considering take part in improvement of Bandit project, contributions are always welcome!  
+Here are guidelines and rules that can be helpful if you plan to want to get involved in the project.
+
+#### Table Of Contents
+[Code of Conduct](#code-of-conduct)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+  * [Things You Should Know Before Getting Started](#things-you-should-know-before-getting-started)
+
+## Code of Conduct
+Everyone who participates in this project is governed by [Bandit Code of Conduct](https://github.com/Glyphack/bandit/blob/master/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct).
+## Reporting Bugs
+If you encountered a bug and want to let us know use [GitHub issues](https://guides.github.com/features/issues/).
+
+**Before submitting a new issue** you might want to check [existsing issue](https://github.com/PyCQA/bandit/issues) to know if there is alredy reported issue and the issue is still open
+in that case you can add a comment to the issue instead of creating a new one.
+
+### Submitting your first issue
+We encourage using a issue template to improve quality of reported issues, However if you feel limited by the template
+You can propose a better issue templte.
+navigate to issues tab and select `new issue`, then select the **bug report** template and fill out the form,
+to submit a good bug report keep in mind that:
+* Use a descriptive title so other people can understand what is this issue about.
+* Be specific about the details, for e.g. what command did you used, what version of Bandit did you use or in what
+environment you observed the bug(CI or development) and other things that may be useful.
+
+## Suggesting Enhancements
+If you Want to suggest and enhancement open a new issue and use the **feature request** template.
+
+**Before submitting an enhancement** please check for existing [feature requests](https://github.com/PyCQA/bandit/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement)
+and make sure the feature does not exist or requested already.
+
+useful things to point out in your feature request:
+* Explain your feature request in a way that everyone can understand maybe with example from another project or pictures
```

Comment:
Explain your feature request in a way that everyone can understand maybe with **an** example from another project or